helsinki_theme/static/css/helsinki_theme.scss

```diff
@@ -0,0 +1,58 @@
+// Font awesome
+$fa-font-path: '../fonts/font-awesome';
+@import 'font-awesome/scss/font-awesome';
+
+// Helsinki brand variables
+@import 'hel-bootstrap-3/src/sass/helsinki-variables';
+@import 'application-variables';
+
+// Bootstrap with custom variables
+@import 'hel-bootstrap-3/src/sass/theme-bootstrap-variables';
+@import 'bootstrap-sass/assets/stylesheets/bootstrap';
+
+// Helsinki Bootstrap overrides
+@import 'hel-bootstrap-3/src/sass/theme-custom-styles';
+
+// Bootstrap Social
+@import 'bootstrap-social/bootstrap-social.scss';
```

Comment:
Needs to be explicitly .scss as the package contains .css file with same name in same directory